t/data/portable_pipelines/ncov2019-artic-nf/cf01166c42a/product_release.yml

```diff
@@ -13,6 +13,7 @@ study:
     portable_pipelines:
       - pp_name: "ncov2019-artic-nf"
         pp_version: "cf01166c42a"
+        pp_repo_url: "https://github.com/connor-lab/ncov2019-artic-nf"
```

Comment:
Is it ok that this URL doesn't reference a particular release? Does pp_version somehow reference a commit?

Reply:
pp_version is either a commit or a tag. For us in production it was always a tag, while in development/testing it's often a commit.

Reply:
Very good

Reply:
POD extended.